src/actions/guides/database/migrations.cr

```diff
@@ -254,15 +254,15 @@ class Guides::Database::Migrations < GuideAction
 
     > You can also use a symbol for a table name. For example `create :users`.
 
-    ## Using fill_existing_with
+    ## Using fill_existing_with and default values
 
-    When using the `add` method inside an `alter` block, there's an additional option `fill_existing_with`.
+    When using the `add` method inside an `alter` block, there's an additional option `fill_existing_with`.  This is useful when a default value is **NOT** needed, but the existing data should be backfilled anyway.
 
-    If your column is required, you will need to set a default value on all records otherwise you'll have errors.
+    If your column is required, you will need to set a **default** value on all records otherwise you'll have errors.  **NOTE:** `default` also back-fills existing data just like `fill_existing_with` and may not be used in combination with the `default` option.
```

Comment:
```suggestion
    If your column is required, you will need to set a **default** value on all records or use `fill_existing_with` otherwise you'll have errors.  **NOTE:** You must use either `default` or  `fill_existing_with`. They can't be used together since they both solve similar problems.
```

The default doesn't really "backfill" data. It's just that you don't need `fill_existing_with` if you have a default since they serve similar pruposes. If you have a default then you don't need fill_existing_with. Does that make sense?